Gemfile.lock

```diff
@@ -138,49 +138,49 @@ GEM
     method_source (1.0.0)
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2020.0512)
+    mime-types-data (3.2020.1104)
     mimemagic (0.3.5)
-    mini_magick (4.10.1)
+    mini_magick (4.11.0)
```

Comment:
### [mini_magick](https://github.com/minimagick/minimagick)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.10.1 → 4.11.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **10** commits. See the full changes on [the compare page](https://github.com/minimagick/minimagick/compare/v4.10.1...v4.11.0).

These are the individual commits:
- (8ce421a) [Fix Imagemagick download URI on CI (#506)](https://github.com/minimagick/minimagick/commit/8ce421a060956eee4ad44322b18717b43ac99af6)
- (13a8835) [Suppress `warning: method redefined; discarding old cli and …](https://github.com/minimagick/minimagick/commit/13a8835f7c860daa2eed206ed7f0a2cf9931f600)
- (c32d09f) [Remove Ruby 1.9.3 from README (#513)](https://github.com/minimagick/minimagick/commit/c32d09f5c35034ba9199a12852abf4802bb05515)
- (986466d) [Fix typo (#520)](https://github.com/minimagick/minimagick/commit/986466d6f35e47e9d38f17a0e77ca0f5c27ed061)
- (eee6cc0) [Parse out Ghostscript warnings in identfication (#522)](https://github.com/minimagick/minimagick/commit/eee6cc0d5a9f451fe6741e3aa9983caadf2987d5)
- (630ec8d) [feat: added helper function to convert array of rgb pixels t…](https://github.com/minimagick/minimagick/commit/630ec8d0e42dae169218258c20b22c1ee1f8da2d)
- (065bfab) [Add YARD example tags; option (#507)](https://github.com/minimagick/minimagick/commit/065bfab7c21f92f29f4f08f5b7f497feac895706)
- (2fcb5f1) [Remove validate_on_write from README](https://github.com/minimagick/minimagick/commit/2fcb5f1327175d0dfbf30a337a52d30fe0d61f91)
- (b7b224f) [Accept hash of options in #stack](https://github.com/minimagick/minimagick/commit/b7b224fdeea946aa435d0f0b804243475dcbd1b6)
- (0217b4d) [Bump to 4.11.0](https://github.com/minimagick/minimagick/commit/0217b4d12a56577c42b37a97e8a9ab90e680ee95)


</details>